components/Card/sub-components/Thumbnail.jsx

```diff
@@ -4,7 +4,7 @@ import styled from 'styled-components';
 
 import { colors } from '../../shared/theme';
 
-const Image = styled.img`
+const Image = styled.img.attrs({ loading: 'lazy' })`
```

Comment:
Doubt: Could this be solved by writing it inline like
```
const Thumbnail = ({ ...props }) => <Image {...props} loading="lazy" />;
```
?

Reply:
the effect is the same.
I tested it both ways...

Reply:
With this inline solution we still have to upgrade the styled components lib?

Reply:
unfortunately, yes!